app.py

```diff
@@ -226,6 +226,7 @@ def populate_shotgun(self, session_id, info):
                                                 run_after_job_id,
                                                 self, 
                                                 "backburner_populate_shotgun",
+                                                info.get("destinationHost"),
```

Comment:
As I've mentionned in the other PR, your function signature needs to be backwards compatible with existing code. One easy way to test it is to make sure the old app still works with the newer engine. :)